tests/llm/test_ask_holmes.py

```diff
@@ -62,6 +62,13 @@ def test_ask_holmes(
     # Set initial properties early so they're available even if test fails
     set_initial_properties(request, test_case, model)
 
+    # Check if --only-setup is set
```

Comment:
@Sheeproid I think this is a mistake and shouldn't be here. I moved it in a recent pr to `check_and_skip_test(test_case)` below which also reports the Skip to braintrust properly

Reply:
Ah i see its the new option you added. Can you move it to the same location please? Its used there for investigate tests too